### PR TITLE
Naive client server implementation for key value service example

### PR DESCRIPTION
**Pull request by yashykt**
Naive client server implementation for key value service example

### CODE REVIEW

File: examples/cpp/keyvaluestore/client.cc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+using keyvaluestore::Request;
+using keyvaluestore::Response;
+using keyvaluestore::KeyValueStore;
+
+class KeyValueStoreClient {
+ public:
+  KeyValueStoreClient(std::shared_ptr<Channel> channel)
+      : stub_(KeyValueStore::NewStub(channel)) {}
+
+  // Requests each key in the vector and displays the value as a pair
```
Comment (hcaseyal) [inline examples/cpp/keyvaluestore/client.cc]: "...displays the key and its corresponding value as a pair"

File: examples/cpp/keyvaluestore/server.cc [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+void RunServer() {
+  std::string server_address("0.0.0.0:50051");
+  KeyValueStoreServiceImpl service;
+
+  ServerBuilder builder;
+  // Listen on the given address without any authentication mechanism.
+  builder.AddListeningPort(server_address, grpc::InsecureServerCredentials());
+  // Register "service" as the instance through which we'll communicate with
+  // clients. In this case it corresponds to an *synchronous* service.
```
Comment (hcaseyal) [inline examples/cpp/keyvaluestore/server.cc]: Nit: "In this case,"

File: examples/cpp/keyvaluestore/client.cc (hunk 1 above)
Comment (yashykt) [reply examples/cpp/keyvaluestore/client.cc]: done

File: examples/cpp/keyvaluestore/server.cc (hunk 2 above)
Comment (yashykt) [reply examples/cpp/keyvaluestore/server.cc]: done

**Comment by yashykt** [comment]
Known issues : #17625

**Comment by yashykt** [comment]
Thanks for reviewing!

**Pull request merged**